src/frontend/data/websocket.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { waitFor } from '@testing-library/react';
+import WS from 'jest-websocket-mock';
+import { modelName } from 'types/models';
+
+import { useVideo } from './stores/useVideo';
+import { videoMockFactory } from 'utils/tests/factories';
+import { initVideoWebsocket } from './websocket';
```

Comment:
```suggestion
import WS from 'jest-websocket-mock';
import { waitFor } from '@testing-library/react';

import { modelName } from 'types/models';
import { videoMockFactory } from 'utils/tests/factories';
import { useVideo } from './stores/useVideo';
import { initVideoWebsocket } from './websocket';
```

Reply:
Nop `@testing` is before `jest-websocket-mock`. I will order the other.